### PR TITLE
DDF-3633 updates integrating docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates the Integrating section of the documentation. 

#### Who is reviewing it? 

@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 

@clockard
@lessarderic
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

Content changes only.

#### Any background context you want to provide?

First in a series of edits to Integrating section to improve audience focus. Sections will be revised / pared down / moved to focus on Integrator needs. 

#### What are the relevant tickets?
[DDF-3633](https://codice.atlassian.net/browse/DDF-3633)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc [hunk 1]
```diff
@@ -3,4 +3,13 @@
 :status: published
 :summary: Introduction to Integrating sections.
 
-Integrators will use the existing applications to support their external frameworks. This section will provide details for finding, accessing and using the components of ${branding}.
+${branding} is structured to enable flexible integration with external clients and into larger component systems.
+
+Federation with ${branding} is primarily accomplished throught <<_endpoints,Endpoints>> accessible through http(s) requests and responses.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc]: `through`

File: distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc [hunk 2]
```diff
@@ -3,4 +3,13 @@
...
+
+If integrating with an existing installation of ${branding}, continue to following sections on endpoints and data/metadata management.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc]: `... to the following ...`

File: distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc]: 👍

File: distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_introduction/integrating-intro.adoc]: 👍

File: distribution/docs/src/main/resources/content/_data/federation.adoc [hunk 3]
```diff
@@ -1,76 +0,0 @@
-:title: Federation
-:type: federationIntro
-:status: published
-:summary: Federation for Integrators
-:order: 00
-
-.Federation
-[ditaa, catalog_architecture_federation, png, ${image-width}]
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_data/federation.adoc]: ❓ did we completely replace or remove these diagrams?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_data/federation.adoc]: I removed them. but I'm open to arguments for putting them back in. I think there might be a case for the second one, if we want to explain it better as it doesn't seem to add much understanding here and I don't know if there's a better place for it elsewhere

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
updated after previous build: build now

File: distribution/docs/src/main/resources/content/_data/federation.adoc (hunk 3 above)
Comment (shaundmorris) [reply distribution/docs/src/main/resources/content/_data/federation.adoc]: I'm good for now, since we haven't gotten a lot of feedback on the diagrams

**Pull request merged**